integration-cli/requirements_unix_test.go

```diff
@@ -11,9 +11,12 @@ import (
 	"github.com/docker/docker/pkg/sysinfo"
 )
 
-// SysInfo stores information about which features a kernel supports.
 var SysInfo *sysinfo.SysInfo
```

Comment:
Any reason why you removed this comment? Or was the intent to un-export it?

Reply:
Yeah, probably intended to unexport this.